src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -1370,6 +1492,15 @@ namespace Microsoft.Management.Deployment
         /// (Installing, Installed) together at the same time.
         PackageCatalogReference CreateCompositePackageCatalog(CreateCompositePackageCatalogOptions options);
 
+        [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 12)]
+        {
+            /// Add a catalog to the Windows Package Catalogs.

```

Comment:
Comment what the progress double value means here.  Include the range of values that should be expected.